app/Controllers/Http/VenuesController.ts

```diff
@@ -0,0 +1,55 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+
+import Venue from 'App/Models/Venue'
+import CreateVenueValidator from 'App/Validators/Venue/CreateVenueValidator'
+import UpdateVenueValidator from 'App/Validators/Venue/UpdateVenueValidator'
+
+export default class VenuesController {
+  // get all venues
+  public async index({ response }: HttpContextContract) {
+    const venues = await Venue.query()
+
+    response.json({ success: true, venues })
+  }
+
+  // get one venue per id
+  public async show({ request, response }: HttpContextContract) {
+    const { id } = request.params()
```

Comment:
ich denke wir brauchen hier auch validations, wenn in path keine uuid drin ist also /venue/fakeuuid dann bekommt man ein 500 error.

Reply:
mh ja. Muss man schauen. Man kommt ja meist auf die Url wenn man irgendwo rauf klickt. ich würde da in den routen irgendwo ein redirect machen. So generell bei id seiten. auch bei events

Reply:
können das auch in nem extra PR machen. Die Events brauchen das ja auch

Reply:
Alles klar